### PR TITLE
Reworked attachments

### PR DESCRIPTION
**Pull request by Tyger-Git**
- Attachments can take in any combination of clipboard images, URLs, screenshots and text.
- /viewattach is now chunked by char count, staying within embed limits. Added embed book functionality.
- Logger adjusted to handle attachments cleanly.

### CODE REVIEW

File: src/commands/tickets/viewattach.js [hunk 1]
```diff
@@ -24,16 +25,86 @@ export default {
...
+            }
+            await interaction.update({ embeds: [allEmbeds[currentPage]], components: [row] });
         });
-        const embed = new EmbedBuilder()
-            .setTitle(`🔗 Ticket Attachments for ticket #${ticket.ticketId} 🔗`)
-            .setDescription(ticketAttachments)
-            .setColor([255, 255, 255]); // White
         
-        await interaction.editReply({ embeds: [embed] });
+        collector.on('end', (collected, reason) => {
```
Comment (Fire-Swan) [inline src/commands/tickets/viewattach.js:84]: collected and reason are declared but never used

**Review by Fire-Swan — CHANGES_REQUESTED** [review]
Check comments

File: src/commands/tickets/viewattach.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import Ticket from '../../schemas/ticket.js';
 import emojis from '../../emojis.json' assert { type: "json" };
-import { EmbedBuilder, ApplicationCommandOptionType } from 'discord.js';
+import { EmbedBuilder, ApplicationCommandOptionType, ButtonBuilder, ActionRowBuilder } from 'discord.js';
+import winston from '../../utils/winston.js';
```
[the remark this answers is not in the document]
Comment (Tyger-Git) [reply src/commands/tickets/viewattach.js:4]: Yeah my IDE snuck that in automatically. Winston is globally declared in index.js, so it doesn't need imported.

File: src/commands/tickets/viewattach.js (hunk 1 above)
Comment (Tyger-Git) [reply src/commands/tickets/viewattach.js:84]: Given this structure of embed book and collector to kill the buttons is common through the project, this was simply a copy/paste that didn't get slimmed down. Good catch.

**Pull request merged**